diracx-client/src/diracx/client/patches/utils.py

```diff
@@ -322,3 +336,121 @@ def __init__(
     @property
     def client_id(self):
         return self._client_id
+
+
+class DeviceFlowErrorResponse(Model):
```

Comment:
Very minor comment:
- `DeviceFlowErrorResponse` could go in `patches.models`
- `Auth/JobsOperations` could go in `patches.operations`